gnovm/pkg/gnolang/op_expressions.go

```diff
@@ -152,17 +159,13 @@ func (m *Machine) doOpStar() {
 			tv.SetUint8(dbv.GetByte())
 			m.PushValue(tv)
 		} else {
-			if pv.TV.IsUndefined() && bt.Elt.Kind() != InterfaceKind {
```

Comment:
this is vestigial; unless interface value, TV is never undefined, it at least has the concrete value set by defaultValue().